package.json

```diff
@@ -117,491 +115,166 @@
         "tailwindcss-highlight.configs": {
           "default": {
             "padding": {
-              "enable": true,
-              "regex": "(?<=[:`'\"\\s])(|-)p(|t|b|r|l|x|y)-[^`'\"\\s]+",
-              "options": {
-                "color": "black",
-                "borderRadius": "0.25rem",
-                "backgroundColor": "rgba(187, 196, 136, 1)"
-              }
+              "regex": "(?<=[:`'\"\\s])(|-)p(|t|b|r|l|x|y)-[^`'\"\\s]+"
```

Comment:
```suggestion
              "regex": "(?<=[:`'\"\\s])(|-)p(|t|b|r|l|x|y)-[^`'\"\\s]+",
              "options": {
                "color": "black",
                "borderRadius": "0.25rem",
                "backgroundColor": "rgba(187, 196, 136, 1)"
```

Reply:
If change it like this, it will only work on padding related classes, which is completely opposite to the purpose of the current PR